### ISSUE TITLE
Potential to making it Pivotal Glossary

### ISSUE THREAD

**Issue by kartiklunkad26** [pivotal-cf/glossary#52]
The website route suggests its a "cf glossary". Do we want to explore making it more inclusive by making it a Pivotal Glossary? Including the data folks, labs and other teams in the organization :)

**Comment by jhamon** [comment]
I would love to make this project more inclusive. There was already somebody using the route http://glossary.cfapps.io/ 

...that's really the only reason for the url cf-glossary. Maybe I can try to figure out who that is and see if we can combine efforts. Or maybe you know already? @kartiklunkad26

**Comment by kartiklunkad26** [comment]
Yeah I think that's a great idea! I have no idea who owns http://glossary.cfapps.io. It might be worth a Pivotalk post!

**Comment by mikfreedman** [comment]
Looks like glossary.cfapps.io is built from: https://github.com/Manifaust/cloud-glossary and is maintained by @manifaust . Looks like there's an opportunity to leverage some synergies and maybe combine efforts? 

FWIW I think this project is great and have already started using the JSON version of the dictionary.

**Comment by jhamon** [comment]
@mikfreedman By json version do you mean http://cf-glossary.cfapps.io/words.json ? How are you using it?

@motevets and I added this as part of our Hack Day project to build a [chrome extension](https://chrome.google.com/webstore/detail/pivotal-glossary/ldjikeaflhaeahnfcloapnfpnmbjloog).

**Comment by mikfreedman** [comment]
Hey, I started writing my own addon here: https://github.com/mikfreedman/pivotal-glossary-addon too!

Let's chat sometime!
